### PR TITLE
Set MASTER_BUILD and NUM_CONTAINERS in 'Create report and post results to BigQuery' step

### PR DESCRIPTION
**Pull request by holdenhinkle**
## Description
Regarding ticket [28644](https://app.zenhub.com/workspaces/vsp-5cedc9cce6e3335dc5a49fc4/issues/department-of-veterans-affairs/va.gov-team/28644)

Set the following env vars in the `Create report and post results to BigQuery` step in the `testing-reports` job:
- `IS_MASTER_BUILD`
- `NUM_CONTAINERS`

This will allow us to add `is_master_build` and `num_containers` columns to `cypress_test_suite_executions` BigQuery table which will allow us to report data regarding Cypress tests run on push (selected tests) and Cypress tests run on merge to master (all tests) and the average of the two in DOMO.

The new columns are being populated as expected in the `cypress_test_suite_executions` BigQuery table. Click through to the ticket noted above for more details.

Related PRs in the `testing-tools-team-dashboard-data` repo:
- https://github.com/department-of-veterans-affairs/testing-tools-team-dashboard-data/pull/51
- https://github.com/department-of-veterans-affairs/testing-tools-team-dashboard-data/pull/52

---

I also changed the name of `MASTER_BUILD` to `IS_MASTER_BUILD`

### CODE REVIEW

**Review by ndsprinkle — APPROVED** [review]

**Review by CBonade — APPROVED** [review]
LGTM!

File: .github/workflows/continuous-integration.yml [hunk 1]
```diff
@@ -468,6 +469,10 @@ jobs:
         id: tests
         run: echo ::set-output name=selected::$TESTS
 
+      - name: Set output of IS_MASTER_BUILD
```
Comment (oseasmoran73) [inline .github/workflows/continuous-integration.yml:472]: If you are already setting it as a global variable for `cypress-test-prep`, why would you need to reset it as an output? Maybe I am missing something? But you should be able to get rid of this step and just have `env.IS_MASTER_BUILD` in output

Comment (holdenhinkle) [reply .github/workflows/continuous-integration.yml:472]: Ohh...

https://docs.github.com/en/actions/reference/environment-variables#about-environment-variables

> You can define environment variables for a step, job, or entire workflow using the jobs.<job_id>.steps[*].env, jobs.<job_id>.env, and env keywords.

File: .github/workflows/continuous-integration.yml [hunk 2]
```diff
@@ -10,9 +10,9 @@ on:
 env:
   CHROMEDRIVER_FILEPATH: /usr/local/share/chrome_driver/chromedriver
 
-concurrency:
-  group: ${{ github.ref != 'refs/heads/master' && github.ref || github.sha  }}
-  cancel-in-progress: ${{ github.ref != 'refs/heads/master' }}
+# concurrency:
```
Comment (oseasmoran73) [inline .github/workflows/continuous-integration.yml:13]: Don't forget about this :)

File: .github/workflows/continuous-integration.yml (hunk 1 above)
Comment (oseasmoran73) [reply .github/workflows/continuous-integration.yml]: Oooh, I didn't know you can access it from `needs.<job>.<job_id>.env`. TIL!!!!

I was saying in the output you could have done 

```
outputs:
  is_master_build: $ {{ env.is_master_build }}
```

But that is a pretty neat trick! Thanks for sharing that doc

Comment (holdenhinkle) [reply .github/workflows/continuous-integration.yml]: I tried the following but couldn't reference that env var:
- `needs.cypress-tests-prep.env.IS_MASTER_BUILD`
- `jobs.cypress-tests-prep.env.IS_MASTER_BUILD`
- `github.workflow.jobs.cypress-tests-prep.env.IS_MASTER_BUIL`

So changed it to this, like you suggested:
```
outputs:
  is_master_build: $ {{ env.IS_MASTER_BUILD }}
```

File: .github/workflows/continuous-integration.yml (hunk 2 above)
Comment (holdenhinkle) [reply .github/workflows/continuous-integration.yml]: Done.

**Pull request merged**